serverless.js

```diff
@@ -74,8 +80,32 @@ class KnativeServing extends Component {
 
   // "private" methods
   async getIstioIngressIp(k8s) {
-    const res = await k8s.readNamespacedService('istio-ingressgateway', 'istio-system')
-    return res.body.status.loadBalancer.ingress[0].ip
+    try {
+      const res = await k8s.readNamespacedService('istio-ingressgateway', 'istio-system')
+      return res.body.status.loadBalancer.ingress[0].ip
+    } catch (error) {
```

Comment:
that's a good first step, but eventually we should be more clever to (a) detect istio installations without loadbalancer and (b) also check for alternative gateways (like gloo or kourier). Especially an autodetect of kourier would be interesting as it becomes a valid alternative (and maybe the default) for Knative (and OpenShift serverless)